src/snowflake/cli/api/entities/application_package_entity.py

```diff
@@ -31,3 +34,20 @@ def action_bundle(self, ctx: ActionContext):
         compiler = NativeAppCompiler(bundle_context)
         compiler.compile_artifacts()
         return bundle_map
+
+    @classmethod
```

Comment:
you don't need the cls param, consider making this static instead.

Reply:
Done. Some methods are `classmethod` in case they need to call other methods. The rest are now `staticmethod`.